<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreatorWidget</class>
 <widget class="QWidget" name="CreatorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeView" name="treeView">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <attribute name="headerCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="headerDefaultSectionSize">
      <number>122</number>
     </attribute>
     <attribute name="headerStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="ModificatorFuselageWidget" name="fuselageModificator" native="true"/>
   </item>
   <item>
    <widget class="ModificatorTransformationWidget" name="transformationModificator" native="true"/>
   </item>
   <item>
    <widget class="ModificatorWingWidget" name="wingModificator" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="applyWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="commitButton">
        <property name="text">
         <string>Commit</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ModificatorTransformationWidget</class>
   <extends>QWidget</extends>
   <header>ModificatorTransformationWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ModificatorWingWidget</class>
   <extends>QWidget</extends>
   <header>ModificatorWingWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ModificatorFuselageWidget</class>
   <extends>QWidget</extends>
   <header>ModificatorFuselageWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
